parsers/EIA.py

```diff
@@ -20,18 +20,189 @@
 from .ENTSOE import merge_production_outputs
 
 EXCHANGES = {
+
+#Old exchanges with old zones, to be updated/removed once clients have had time to switch
     'MX-BC->US-CA': 'EBA.CISO-CFE.ID.H',
     'US-BPA->US-IPC': 'EBA.BPAT-IPCO.ID.H',
     'US-SPP->US-TX': 'SWPP.ID.H-EBA.ERCO',
     'US-MISO->US-PJM': 'EBA.MISO-PJM.ID.H',
     'US-MISO->US-SPP': 'EBA.MISO-SWPP.ID.H',
     'US-NEISO->US-NY': 'EBA.ISNE-NYIS.ID.H',
-    'US-NY->US-PJM': 'EBA.NYIS-PJM.ID.H'
+    'US-NY->US-PJM': 'EBA.NYIS-PJM.ID.H',
+
+#Exchanges to non-US BAs
+    'US-CAL-CISO->MX-BC': 'EBA.CISO-CFE.ID.H', #Unable to verify if MX-BC is correct
```

Comment:
given the comment, is there any action item here? Something we should check before merging?

Reply:
I tried to find more information on which Mexican BAs had exchanges, but was unable to find it on the Mexico side. Since we don't have any mexican production data at this point, I don't think this will affect anything yet, this is more a note for the future, if we find a source for mexican data we should check these exchanges too.